.github/workflows/build-plugin-zip.yml

```
@@ -326,7 +326,7 @@ jobs:
                   git config user.name "Gutenberg Repository Automation"
                   git config user.email gutenberg@wordpress.org
 
-            - name: Setup Node (for CLI)
+            - name: Setup Node.js (for CLI)
```

```suggestion
            - name: Setup Node.js
```

I don't think we need for CLI here.